### DISCUSSION TITLE
Is there any way to print '\b' with Rich?

### DISCUSSION THREAD

**Discussion by CKylinMC** [Textualize/rich#1230] [Q&A] [answered]
If I want to print '\b' with Rich, what should I do? 
For example:
My code without Rich:
```python
print("Something here ???", end='', flush=True)
print("\b\b\b!!!")
```
It should print:
```
Something here !!!
```
But when I using Rich:
```python
from rich import print
print("Something here ???", end='', flush=True)
print("\b\b\b!!!")
```
I got:
```
Something here ???!!!
```

Is it an unsupported feature?

**Comment by willmcgugan** [comment, marked as answer]
Rich will strip out some control codes because they break formatting. If you want dynamically updating output have a look at Status or Live in the docs.

If want to bypass rich and write directly to stdout you can call `console.file.write`

**Reply by CKylinMC** [reply]
Well, thanks!
